src/main_class.cpp

```diff
@@ -160,20 +160,25 @@ void CMain::loadTranslation() {
 	QString src_app_trans_path = bin_path + QLatin1String("/data/translations");
```

Comment:
I had to change this path:
```suggestion
	QString src_app_trans_path = bin_path + QLatin1String("/translations");
```

Reply:
Yes, makes sense.

I finally figured out why I couldn't get the translations to work. Me stupid of course assumed Qt would honor standard locales, i.e. something like `LANG=de_DE.UTF-8` would work. But no, no, no, you have to set `LANGUAGE=`. 

I verified that translations work with plain build and AppImage.